File: src/sdk-reference/cpp/1/kuzzle/add-listener/snippets/add-listener.cpp
{
  using kuzzleio::SharedEventListener;
  using kuzzleio::EventListener;

  SharedEventListener listener = std::make_shared<EventListener>(
    [](const std::string &payload) {
      std::cout << payload << std::endl;
    });

  SharedEventListener other_listener = std::make_shared<EventListener>(
    [](const std::string &payload) {
      std::cerr << payload << std::endl;
    });

  kuzzle
    ->addListener(kuzzleio::KUZZLE_EVENT_CONNECTED, listener)
    ->addListener(kuzzleio::KUZZLE_EVENT_CONNECTED, other_listener);
}

std::cout << "Listener successfully added" << std::endl;

File: src/sdk-reference/csharp/1/kuzzle/add-listener/snippets/add-listener.cs
EventListener listener = [](const string payload) {
  Console.WriteLine(payload);
};
EventListener other_listener = [](const string payload) {
  Console.Error.WriteLine(payload);
};

kuzzle.addListener(CONNECTED, listener).addListener(CONNECTED, other_listener);

Console.WriteLine("Listener successfully added");
